src/components/Header/Header.story.jsx

```diff
@@ -49,6 +49,7 @@ const HeaderProps = {
   url: 'http://localhost:8989',
   className: 'custom-class-name',
   appName: 'Watson IoT Platform ',
+  skipto: 'skip',
```

Comment:
Is this used or simply passed through to the Carbon component? Does it need to be in proptypes/defaultprops?

Reply:
this is listed in the proptype defs